Core/Utilities/Time.py

```diff
@@ -50,10 +50,17 @@ def timed( *args, **kw ):
     result = method( *args, **kw )
     te = nativetime.time()
 
+    pre = dt.utcnow().strftime( "%Y-%m-%d %H:%M:%S " ) + 'UTC '
+
     try:
-      pre = str( dateTime() ) + args[0].log._systemName + args[0].transString
-    except Exception:
-      pre = str( dateTime() )
+      pre += args[0].log._systemName + '/' + args[0].log._subName + '   TIME: ' + args[0].transString
+    except AttributeError:
+      try:
+        pre += args[0].log._systemName + '    TIME: ' + args[0].transString
+      except AttributeError:
+        pre += args[0].log._systemName + '/' + args[0].log._subName + '   TIME: '
+    except IndexError:
+      pre += '  TIME : '
 
     print( "%s Exec time ===> function %r arguments len: %d -> %2.2f sec" % ( pre, method.__name__, len( kw ), te - ts ) )
```

Comment:
'arguments len' is not really explanatory... what about 
'(nb of named args: %d)' 
or 
'(%s named args)' % len(kw) if kw else 'no'
named to also be keyword if you prefer ;-)
What would be more interesting is the number of items the method is dealing with but we have no convention that the subject of the method is for example the first argument... Could we have a way to tell timeThis() how many "items" (i.e. files in most cases) the method is working on?